src/OverflowScroll/data/useOverflowScrollActions.js

```diff
@@ -15,7 +15,24 @@ const useOverflowScrollActions = ({
   scrollAnimationBehavior = 'smooth',
   onScrollPrevious,
   onScrollNext,
+  onChangeOffset,
+  currentOffset,
+  offset,
 }) => {
+
+  const getOffset = (offsetString, maxOffset) => {
+    let offset = 0;
+
+    if (offsetString.endsWith("px")) {
+      offset = parseInt(offsetString);
+    } else if (offsetString.endsWith("%")) {
```

Comment:
[suggestion] Would it be helpful to potentially treat the `offset` value as a distinct argument than the `offset` type (pixel or percent). For example, something like:

```jsx
useOverflowScrollActions({
  offset: 25, // number or string
  offsetType: 'fixed', // "percentage" or "fixed" (default)
});
```

This way, this logic wouldn't need to have conditionals based on `px` or `%` in a string itself.

Curious to hear your thoughts :)